README.md

```diff
@@ -67,42 +76,58 @@ remains as a reference, but is not recommended for production deployments.
 # Building Pelikan
 
 ## Requirement
+
 - Rust [stable toolchain](https://www.rust-lang.org/learn/get-started)
+- (to build `momento_proxy`) Follow [the gRPC instructions here][grpc-url]
+  to install Protocol Buffer Compiler. Alternatively, you can skip the
+  `momento_proxy` package.
+
+_To build legacy Pelikan, you will need:_
```

Comment:
I think these are needed for the Rust build too? I believe the boringssl dependency requires that we have some basic C toolchain stuff. Might be good to start with a minimal Debian or fresh MacOS vm and note the steps required.

Reply:
I created a reference Dockerfile from `debian-slim` in this PR https://github.com/pelikan-io/pelikan/pull/80/files#diff-00fa8723a488e725797cc5951c4d40838b610678337ffd034cd8306132f3e704 that could potentially be used in CI to make sure we have all our required packages listed. Otherwise, we can either extract it into non-executable documentation or keep the reference file around

Reply:
That's awesome, thanks @hderms !